src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1248,7 +1248,10 @@ trait Contexts { self: Analyzer =>
      */
     private[Contexts] def importedAccessibleSymbol(imp: ImportInfo, sym: => Symbol): Symbol =
       if (isExcludedRootImport(imp)) NoSymbol
-      else sym.filter(isAccessible(_, imp.qual.tpe, superAccess = false))
+      else sym.filter { s =>
+        if (s.isTopLevel) s.initialize
```

Comment:
Forcing makes me a bit nervous..

We could call `sym.exists` instead, which only completes symbol loaders, not lazy types. Though it also completes `UnPickler$Scan$LazyTypeRef`, not sure why that is. IIUC, we only would need to complete `SymbolLoader`s.

In the test case we get `SymbolLoaders$ClassfileLoader` --(.exists)--> `UnPickler$Scan$LazyTypeRef` --(.exists)--> (`Types$ModuleTypeRef`).

Reply:
This was inspired by your `exists` trick, I understood the completer, but thought `initialize` was equivalent. I can try detuning to `exists`. And check whether it matters what is imported.

Reply:
Here's a spot in the compiler where we call `load` twice: https://github.com/scala/scala/blob/0fd61f7eae54ae79b0713a40d8f2ef9e0962064f/src/reflect/scala/reflect/internal/Symbols.scala#L1799-L1805